### DISCUSSION TITLE
Page for adding additional proof does not allow uploading images

### DISCUSSION THREAD

**Discussion by acheong0** [community/community#17712] [GitHub Education]
The only option given is to use my camera which is extremely low resolution and all images submitted have been deemed too blurry.

**Comment by mlarasusan** [comment]
Hello 👋

You can contact our support team here 👉 https://support.github.com/contact/education  [✨](https://emojipedia.org/sparkles/)

**Reply by TungusSs** [reply]
Hello! The support service responds with a template response that does not give any specifics on how to fix it or what is better to do so that your application is not rejected using a bot 30 seconds after you have submitted the application

**Comment by elisemoe** [comment]
Hi @acheong0  👋 sorry for the frustration! Would you be able to apply from a phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.

**Reply by acheong0** [reply]
I have no phone :(

**Comment by sharklayer** [comment]
None of these solutions work, it seems the only want to beg until they get bored and decide to approve. I've been applying for two weeks now, that's just insane and ridiculous...

**Comment by mlarasusan** [comment]
Sorry about that. Did you follow the instructions in this [blog](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)? They may be helpful.

**Comment by acheong0** [comment]
A few issues with my school:
- GitHub is blocked occasionally in China. I must use a proxy or VPN to access the internet. (Though I applied today without issues without VPN)
- My School ID card is very old and parts of it are faded
- I asked the school for a certificate but my student ID was not included.
- I'm not sure we even have an ID system other than on our Lunch card to charge us money
